src/intl/messages.js

```diff
@@ -62,6 +62,13 @@ export const messages: { [messageId: string]: MessageType } = {
   cancel: 'Cancel',
   cannotUpdateCloudInitHostname: 'Host name cannot be synchronized with VM name because the VM name is not a valid host name.',
   cantLogonToConsole: 'Single sign on failed. Please check to see if the guest agent is running on your virtual machine. Contact your administrator if the problem continues.',
+  cantOpenConsole: "Can't open console: {message}",
```

Comment:
when should this message appear? it's not informative and seems redundant.

Reply:
![image](https://user-images.githubusercontent.com/64131213/109476036-1bc0fd80-7a7f-11eb-9e30-655d47af3613.png)